src/normalize/entities.ts

```diff
@@ -441,11 +445,43 @@ export const Appointment = new Entity({
 	inst: ['instances', Instance]
 });
 
+export const Document = new Entity({
+	id: 'id',
+	...commonFieldForMessageAndDocuments,
+	uuid: 'uuid',
+	name: 'name',
+	luuid: 'folderUuid',
+	mdver: 'metadataVersion',
+	meta: 'metaData',
+	descEnabled: 'descEnabled',
+	ver: 'version', //same item may have different versions (i.e same names) will need to implement ListDocumentRevisionsRequest
+	leb: 'lastEditedAccount',
+	cr: 'revisonCreator',
+	cd: 'revisedCreationDate',
+	loid: 'lockOwnerId',
+	ct: 'contentType'
+});
+
+export const UploadDocumentId = new Entity({
```

Comment:
we don't need entities if key and value are same

Reply:
Done